src/index.js

```diff
@@ -376,4 +379,4 @@ exports.RecordRemover = require('./services/exposed/record-remover');
 exports.RecordSerializer = require('./services/exposed/record-serializer');
 exports.PermissionMiddlewareCreator = require('./middlewares/permissions');
 
-exports.PUBLIC_ROUTES = ['/', '/sessions', '/sessions-google'];
+exports.PUBLIC_ROUTES = ['/', '/sessions', '/sessions-google', '/healthcheck'];
```

Comment:
[Code style] Should we keep the routes declared in the alphabetical order?
```suggestion
exports.PUBLIC_ROUTES = ['/', '/healthcheck', '/sessions', '/sessions-google'];
```